l10n/en/products/vpn/more/ip-address.ftl

```diff
@@ -20,7 +20,7 @@ vpn-ip-address-does-it-travel = Does it travel with you?
 
 # Variables
 #   $url (url) - https://www.mozilla.org/products/vpn/more/when-to-use-a-vpn/
-vpn-ip-address-no-your-ip = No. Your IP address is only associated with one location unless you are <a href="{ $url }">using a VPN</a> (we will get more into that later). When you are at your home and connecting to the internet you pay for, you are using one. However, if you check your email at home in the morning, then scan the news at a local coffee shop while waiting for your coffee, and then work from an office, you will have used different IP addresses at each location.
+vpn-ip-address-no-your-ip = No. Your IP address is only associated with one location unless you are using a VPN (we will get more into that later). When you are at your home and connecting to the internet you pay for, you are using one. However, if you check your email at home in the morning, then scan the news at a local coffee shop while waiting for your coffee, and then work from an office, you will have used different IP addresses at each location.
```

Comment:
If we're removing a link from a string here, then in order for this to be picked up by translators we should use a new string ID here instead e.g. `vpn-ip-address-no-your-ip-v2`. This will also then need to be updated in the template.

Reply:
@alexgibson Updated - take a look at https://github.com/mozilla/bedrock/pull/11225/commits/e1545c4f3e593a52c789d108930e89bc3dabf3ed whenever you get a mo